selfdrive/car/body/values.py

```diff
@@ -26,6 +26,12 @@ class CAR:
   CAR.BODY: CarInfo("comma body", package="All"),
 }
 
+FINGERPRINTS = {
+  CAR.BODY: [{
+    513: 8, 516: 8, 514: 3, 515: 4,
```

Comment:
The full fingerprint appears to be `{513: 8, 516: 8, 514: 3, 515: 4, 1833: 8, 1832: 8}`

Reply:
Though those two higher addrs are only sent three times, but close to the start of the route.